build_in_functions_task_min_max_func.py

```diff
@@ -0,0 +1,65 @@
+# Implement your own implementation of function max and min
+# (* additional argument amount of result, if you pass 2, function should return 2 max values from the list)
+from typing import Callable
+
+filtered_min_list = []
+filtered_max_list = []
```

Comment:
why these variables are out of function? and why do you need list at all? You need to store only 1 value